docs/dashboards/chart-builder.md

```diff
@@ -58,6 +58,20 @@ On the list of metrics you can find a special metric `count()`. This
 metric is available for [Logs](https://sematext.com/docs/logs/) and represents the number
 of matching logs.
 
+### Only latest interval
+
+Only Latest Interval toggle is available in [bar](./bar-pie-donut-chart), [pie & donut](./bar-pie-donut-chart), [geomap](./geomap), and [numeric](./numeric-component) components. When it is toggled on, only the latest value of the selected metric is displayed within the specified time range.
+
+![Chart builder metrics](../images/dashboards/chart-builder-latest-interval.png)
+
+If the  [Group By](https://sematext.com/docs/dashboards/chart-builder/#group-by) option is selected when only latest interval toggle is on, the displayed value will depend on the selected group by action.
+
+- **All Separetly:** Shows the latest value of the selected metric for each group. For instance, if you choose to group data by os.host and there are multiple `os.host` categories, the latest value of the selected metric for each distinct `os.host` will be displayed.
```

Comment:
Separetly => Separately
Should first os.host be surrounded by ticks?
multiple `os.host` categories ==> multiple `os.host` values?